lib/tasks/colormaps.rake

```diff
@@ -35,6 +35,19 @@ namespace :colormaps do
 
     exit 1 if error_count > 0
 
+    # explicitly invoke the task here to make sure task colormaps:load completes successfully.
+    Rake::Task['colormaps:log_colormaps_load'].invoke
```

Comment:
Would it be useful to distinguish between the cases of running but failing and not running at all?